src/frontend/efiling-frontend/src/domain/package/package-confirmation/PackageConfirmationService.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import FileSaver from "file-saver";
+
+/**
+ * Retrieves the Uploaded Document from the backend API for the given submissionId and filename.
+ *
+ * @param submissionId the submissionId associated with the package
+ * @param file the document to retrieve. Shape must be { mimeType: "string", documentProperties: { name: "string" } }
+ */
+export const downloadFile = async (submissionId, file) => {
```

Comment:
Should this go in the document service ?

Reply:
Yes, that's a good thought. I'll get on this.